addons/binding/org.openhab.binding.wizlighting/src/main/java/org/openhab/binding/wizlighting/internal/entities/Param.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.wizlighting.internal.entities;
+
+/**
+ * This POJO represents an abstract Request Param
+ *
+ * @author Sriram Balakrishnan - Initial contribution
+ *
+ */
+public abstract class Param {
```

Comment:
If this class in empty, i.e. with no implementation code, would it not be sufficient to change it to an `interface` instead?